lib/assets/javascripts/turbolinks.coffee

```diff
@@ -33,7 +33,7 @@ fetch = (url, options = {}) ->
     document.location.href = url.absolute
     return
 
-  if options.change or options.keep
+  if options.change or options.append or options.prepend or options.keep
```

Comment:
We do `options.change or options.append or options.prepend` in many places now, so let's add a helper function. E.g. `isPartialReplacement(options)`.